src/config/config.js

```diff
@@ -1,3 +1,7 @@
+export const version ={release: 1, name:'1.0.0beta'};
+export const chainList=[
+    { short: 'BTS', name: 'BitShares'}
```

Comment:
I think `chain_id` should be added throughout as well to avoid any counterfeiting or accidental use.

Reply:
agreed